### PR TITLE
Header fix

### PR DESCRIPTION
**Pull request by roo-tech**
copied header and added js references to all html files except event-detail html

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -309,8 +308,9 @@ <h4>Land Acknowledgement</h4>
 	<div class="overlay-la hidden"></div>
 
 	<!-- JavaScript -->
-	<script src="/js/footer-modal.js"></script>
-	<script src="/js/main.js"></script>
+	<script src="js/footer-modal.js"></script>
+	<script src="js/main.js"></script>
+	<script src="./js/header.js"></script>
```
Comment (MaidenManitoba) [inline about.html]: please remove the ./ in the js/header.

File: about.html [hunk 2]
```diff
@@ -15,101 +15,100 @@
 </head>
 
 <body>
-	<!-- HEADER -->
```
Comment (MaidenManitoba) [inline about.html:18]: Looks good besides the hamburger button issue?

**Review by MaidenManitoba — COMMENTED** [review]
It's looking good, you just need to address the hamburger button issue.

**Review by samifa308 — COMMENTED** [review]
its looking good!

File: about.html [hunk 3]
```diff
@@ -15,101 +15,100 @@
...
+				  <a href="/events.html"> Events</a>
+				</li>
+				<li>
+				  <a href="/team.html"> Our Team</a>
+				</li>
+			  </ul>
+			</div>
+			<a href="/contact.html" class="button2"> Contact Us</a>
+			<div id="menu-container">
+			  <button id="mobile-menu">
```
Comment (maxharrisnet) [inline about.html]: This is still giving an error with the WAVE tool. I think we just need to add some descriptive text that is only visible to screen readers. Feel free to add an `.sr-only` class if we don't have one yet. 

Screen Shot 2024-01-30 at 8 07 38 AM

File: about.html (hunk 1 above)
Comment (maxharrisnet) [reply about.html]: In VS code, you can do a find/replace over multiple files: https://blog.devsense.com/2022/find-and-replace-with-regexp

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
Looks good, but we need to fix the button issue that comes up with the WAVE tool.

File: about.html (hunk 1 above)
Comment (roguemacaroon) [reply about.html]: removed . infront of /js in all html files and resolved this issue

File: about.html (hunk 2 above)
Comment (roguemacaroon) [reply about.html:18]: resolved this issue in all html pages and added in the sr-only class to global variables

File: about.html (hunk 3 above)
Comment (roguemacaroon) [reply about.html]: resolved this issue

**Comment by maxharrisnet** [comment]
@roguemacaroon I just fixed some merge conflicts on this branch. You will need to do a git pull.

**Review by maxharrisnet — APPROVED** [review]
Awesome work!

**Pull request merged**